external/os-guides/pentesting/kali.md

```diff
@@ -122,11 +122,19 @@ $ qvm-copy kali-key.asc
 # echo 'deb https://http.kali.org/kali kali-rolling main non-free contrib' > /etc/apt/sources.list
 ```
 
-8. Update the template 
+8. Replace gcc8
+
+```shell_session
+# apt-get remove libgcc-8-dev && apt-get install libc6-dev
+```
+
+**Note:** This step is currently needed. But it will not always be the case.
```

Comment:
This is a bit cryptic. Do you mean something like, "This step is required until X is fixed/implemented"?

Reply:
Something like that yes, with more words: 
This kind of issues will popup and disappear without notice.
It depends on the differences of dependencies between the kali repository, the testing qubes repository and the dependencies issues related to the upgrade from stable debian to testing debian/kali.

In comparison, the others steps are required regardless of the dependencies version of kali & qubes repository.

IDK how to make that easily understandable

Reply:
Ok, thanks for the explanation. I will provide an edit that attempts to make it clearer.

Reply:
Actually, I'm not sure whether "this step" refers to step 8 or step 9.

Reply:
I also don't know what you're referring to what you say "this kind of issue."

I suggest that you add the type of more verbose explanation that you provided here.

Reply:
Re-worded this part to be more specific